deployments/examples/ocis_wopi/docker-compose.yml

```diff
@@ -127,6 +127,9 @@ services:
     logging:
       driver: "local"
     restart: always
+    depends_on:
+      collabora:
+        condition: service_healthy
```

Comment:
This is only part of V2, right? V3 https://docs.docker.com/compose/compose-file/compose-file-v3/#depends_on doesn't mention it.